stock_move_backdating/__openerp__.py

```diff
@@ -19,23 +19,18 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+
+
 {
-    "name": "Stock Move Backdating",
-    "version": "1.0",
-    'author': ['Marco Dieckhoff, BREMSKERL', 'Agile Business Group'],
-    "category": "Stock Logistics",
+    'name': 'Stock Move Backdating',
+    'version': '1.0',
+    'category': 'Stock Logistics',
+    'author': 'Bremskerl, Odoo Community Association (OCA)',
```

Comment:
author is here https://github.com/OCA/stock-logistics-workflow/pull/142/files#diff-8ebcf17df81968ac09b007cf5d372958R28